control-plane-operator/controllers/hostedcontrolplane/kas/auth_types.go

```diff
@@ -13,277 +13,46 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 type AuthenticationConfiguration struct {
```

Comment:
You want this copy https://github.com/openshift/kubernetes/blob/release-4.15/staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go with the json tags